MyFirstApp/MyFirstApp/ViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ViewController.swift
+//  MyFirstApp
+//
+//  Created by Ivan Tyurin on 22.01.2020.
+//  Copyright © 2020 Ivan Tyurin. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var iosNum = Int.random(in: 0...100)
+    
+    @IBOutlet weak var cheat: UILabel!
+    @IBOutlet weak var mainText: UILabel!
+    @IBOutlet weak var hiddenBtn: UIButton!
+    @IBOutlet weak var userText: UITextField!
+    @IBOutlet weak var gameBtnStates: UIButton!
+    
+    @IBAction func gameBtn(_ sender: UIButton) {
+        let userNum = Int(userText.text!)!
+        if  userNum == iosNum || userNum == 42 {
```

Comment:
why 42 ?

Reply:
Because, this is the answer to the Ultimate question of life, the universe and Everything :-)
It's just a joke, I’ll delete it in the final version, with all the additional tasks.